glean-core/android/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,2 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="org.mozilla.gleancore" />
+    package="mozilla.telemetry.glean" />
```

Comment:
This name should match the name of the package.  Needed for `BuildConfig` to work.  This also puts the generated metrics in the correct namespace.